### PR TITLE
Readme file Updated

### PR DESCRIPTION
**Pull request by snehaM2210**
Solves the issue #57 
image
image
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+</ul>
+
+<h2>Software Used💻</h2>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+  </ul>
+<h2>Screenshots📸</h2>
+<img src="static\images\Js-gamesCollage.png">
```
Comment (geekymeeky) [inline README.md:36]: The image isn't rendering properly. Modify the src to `./static/images/Js-gamesCollage.png`.

**Review by geekymeeky — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (snehaM2210) [reply README.md:36]: Okay, sure sir.

**Comment by geekymeeky** [comment]
@snehaM2210 You can add more commits by pushing to your feature branch. Closing this PR isn't required.

**Comment by snehaM2210** [comment]
Yes sir... I have done the changes... Do tell me if some other changes are required.

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+</ul>
+
+<h2>Software Used💻</h2>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+  </ul>
+<h2>Screenshots📸</h2>
+<img src="./static/images/Js-gamesCollage.png">
```
Comment (geekymeeky) [inline README.md:36]: `alt`  attribute is required for the image tag. Add `alt="collage preview"` attribute to the image tag

**Review by geekymeeky — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (snehaM2210) [reply README.md:36]: Okayy..doing it now.

**Review by geekymeeky — APPROVED** [review]

**Pull request merged**